src/components/CommunityBanner.tsx

```diff
@@ -22,7 +22,7 @@ const CommunityBanner: React.FC<Props> = ({ className, text, title, bg }) => {
     >
       <div className="flex flex-col gap-3 md:w-[60%] h-full pl-4 pr-16 pb-10 pt-44 md:p-12 text-white items-start justify-end">
         {title && (
-          <h1 className="font-semibold text-3xl font-aktiv">{title}</h1>
+          <h1 className="font-semibold text-[32x] leading-10 font-aktiv">{title}</h1>
```

Comment:
You're missing a `p` in `px`

Reply:
🙏 thank you for spotting that typo! it's fixed! 😁